test/js/node/http2/node-http2-memory-leak.js

```diff
@@ -32,8 +32,8 @@ async function runRequests(iterations) {
   for (let j = 0; j < iterations; j++) {
     let client = http2.connect(info.url, { rejectUnauthorized: false });
     let promises = [];
-    // 100 multiplex POST connections per iteration
-    for (let i = 0; i < 100; i++) {
+    // 10 multiplex POST connections per iteration
```

Comment:
why not 100?

Reply:
```suggestion
    // 100 multiplex POST connections per iteration
```

Reply:
I think 100 is too much, we can up for 500 connections instead of 50 connections and would be more useful for detecting leaks than 100 multiplex, also we check for max 1 body leak per connection.